### PR TITLE
Feat(Home Screen):Adds a list of seen phrases

### PR DESCRIPTION
**Pull request by Voninkazo**
Hello Team,

We have added seen phrases in the home screen for users to go back and learn phrases that they get wrong and store them in AsyncStorage. We think we have finished it but you are welcome to review it and give any comments.
You can find the branch [here](https://github.com/onja-org/malagasy-rn-Lead/tree/seen-phrases)
 Refs: #35bj9m

Thanks in advance!

### CODE REVIEW

File: Root.js [hunk 1]
```diff
@@ -19,6 +20,7 @@ const Root = () => {
   }, [toggleStorybook]);
 
   // return storybookActive ? <Storybook /> : <App />;
```
Comment (tsipoy) [inline Root.js]: Could you remove the import storybook and the return storybookActive comments.

File: src/screens/Home.js [hunk 2]
```diff
@@ -61,6 +62,15 @@ export default ({
     navigation.navigate('Learn');
   };
 
+  const openSeenPhrases = () => {
+    if (seenPhrases.length !== 0) {
+      setPhrases(seenPhrases);
+      navigation.navigate('Learn');
+    } else {
+      return null;
+    }
```
Comment (andrzej-onja) [inline src/screens/Home.js:69]: You dont need 

`else {
      return null;
    }`

File: src/screens/Learning.js [hunk 3]
```diff
@@ -50,10 +53,21 @@ export default ({
...
+        const correctPhraseInSeenPhrases = seenPhrases.find(
+          phr => phr.id === item.id,
+        );
+        // update seen phrases
+        if (correctPhraseInSeenPhrases) {
+          updateSeenPhrases(correctPhraseInSeenPhrases);
+        }
       } else {
         // TODO add to seen
+        // add seen phrases
```
Comment (andrzej-onja) [inline src/screens/Learning.js]: remove TODO you just implemented it

**Review by tsipoy — APPROVED** [review]

**Review by Tanosimboangy — APPROVED** [review]

File: src/redux/constants/index.js [hunk 4]
```diff
@@ -5,3 +5,5 @@ export const SET_LANGUAGE_NAME = 'SET_LANGUAGE_NAME';
 export const SET_CURRENT_CATEGORY = 'SET_CURRENT_CATEGORY';
 export const SET_USER_PHRASES = 'SET_USER_PHRASES';
 export const NEW_KEY_PHRASES = '@NewPhrases';
```
Comment (andrzej-onja) [inline src/redux/constants/index.js]: NEW_KEY_PHRASES  and SEEN_PHRASES_KEY should be moved to src/utils/storage.js

**Review by andrzej-onja — APPROVED** [review]

**Pull request merged**